platform/administer/templates/index.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: Templates
+sidebar_label: Templates
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Flow, { Step } from "@site/src/components/Flow";
+import Button from "@site/src/components/Button";
+import Label from "@site/src/components/Label";
+import NavStep from "@site/src/components/NavStep";
+import PartialVirtualClusterCreateOptionalTemplateUI from "../../_partials/vcluster/create-ui-optional-fields-template.mdx";
+
+
+# Templates
+
+vCluster templates define reusable configurations for creating virtual Kubernetes clusters. They allow platform teams to standardize cluster creation while giving users a controlled way to customize certain settings. Templates are implemented using Kubernetes Custom Resources (`VirtualClusterTemplate`) and can be managed with tools like `kubectl`, Helm, or Terraform.
```

Comment:
I think I would mention Argo CD (especially since we have an integration) and not sure about Terraform as it isn't K8s native like Argo CD.